styles.css

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+}
+
+/* Navbar */
+.flex-Nav {
+  display: flex;
+  justify-content: space-between;
+  background: #3c3a39;
+  position: absolute;
+  padding: 0 5px 0 5px;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.nav-Button {
+  background-color: #3c3a39;
+  border: 0;
+  width: 14px;
+  flex-basis: 40px;
+  padding: 8px;
+  border-radius: 8px;
+}
+
+.nav-Button:hover {
+  background-color: black;
+}
+
+.header-Name {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  padding: 10px 12px;
+  margin: 0;
+}
+
+/* headline section */
+#headline {
+  margin: 44px 0 0;
+  height: 590px;
+  display: flex;
+  background: black url(images/Illustration-header.svg) 120px 120px no-repeat;
```

Comment:
- [ ] Great job implementing this image as a background, but in order to match the Figma design better, you should consider making it a bit smaller. Right now your image is too big making the text hard to read.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143365594-3496c080-a59c-40f5-83f0-a012c2e71f4c.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/143365645-27379cbf-7168-44f2-9c92-bb3776f74b2a.png)

Reply:
Fixed BG color

Reply:
and Size